### PR TITLE
Finally my CROD are ready , please review it , I have been waiting for your feedback.

### PR DESCRIPTION
**Pull request by BrayanRosas**

### CODE REVIEW

File: config/room.json [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
     "displayName" : "Floor1Room1",
-    
-    "resources" : [
+
+    "resources" : {
+        "roomsAsoc":
+
             {
-            "quantity" : 1   
+           "resourceId": "1", "quantity": 1
```
Comment (AlejandroJeanCarlo) [inline config/room.json]: FIX THIS

File: tests/CRUD/rooms.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+var init = require('../../init');
+var config = require(GLOBAL.initialDirectory+'/config/config.json');
+var resourceConfig = require(GLOBAL.initialDirectory+config.path.resourceConfig);
+var expect = require('chai').expect;
+var tokenAPI = require(GLOBAL.initialDirectory+config.path.tokenAPI);
+var roomManagerAPI = require(GLOBAL.initialDirectory+config.path.roomManagerAPI);
+var endPoints = require(GLOBAL.initialDirectory+config.path.endPoints);
+var util = require(GLOBAL.initialDirectory+config.path.util);
+var roomJson= require(GLOBAL.initialDirectory+'/config/room.json');
+var mongodb=require(GLOBAL.initialDirectory+'/lib/mongodb');
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/rooms.js]: don't hard code please

File: tests/CRUD/rooms.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+						.and.be.equal(room._id.toString());
+						done();
+			});
+		
+	});
+
+
+	it.skip('CRUD-PUT /rooms/{:roomId} api returns the rooms modified', function (done) {
+
+		var json2=roomJson.roomQueries.roomPut;
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/rooms.js]: what is json2?

File: tests/Smoke/rooms.js [hunk 4]
```diff
@@ -32,7 +32,7 @@ describe('Smoke Testing for Room routes', function() {
 			.getToken(function(err,res){
 				token = res.body.token;
 				endPoint=config.url+endPoints.rooms;
-				json=roomJson.roomQueries[0];
+				json=roomJson.roomQueries.customDisplayName;
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/rooms.js]: what is json?

File: tests/Smoke/rooms.js [hunk 5]
```diff
@@ -77,8 +77,8 @@ describe('Smoke Testing for Room routes', function() {
  * Smoke Test to the service room with the method put for modify the
  * display name of the room 
  */
-	it('PUT /rooms/{roomId}, Verify the status 200',function(done){			
-		json=roomJson.roomUpdateQ[0];
+	it('PUT /rooms/{roomId}, Verify the status 200',function(done){	
+		json.customDisplayName='ChangedByAPI';
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/rooms.js]: same

File: tests/Smoke/rooms.js [hunk 6]
```diff
@@ -97,6 +97,7 @@ describe('Smoke Testing for Room Resources routes ', function() {
 	 * Get a room randomly and create a resource
 	 */
 	before('Before Set',function (done) {
+		//console.log('11111111'+JSON.stringify(json));
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/rooms.js]: remove

File: tests/Smoke/rooms.js [hunk 7]
```diff
@@ -107,8 +108,8 @@ describe('Smoke Testing for Room Resources routes ', function() {
 						json=util.getRandomResourcesJson(resourceConfig.resourceNameSize);
 							roomManagerAPI.post(token,endPoint2,json,function(err,resourceRes){
 								resource=resourceRes;
-								 	endPoint=endPoint+'/'+room._id+'/resources';		
-									json=roomJson.roomUpdateQ[3];
+								 	endPoint=endPoint+'/'+room._id+'/resources';	
+								 	json=roomJson.resources.roomsAsoc;
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/rooms.js]: same

File: config/room.json (hunk 1 above)
Comment (BrayanRosas) [inline config/room.json]: It already was fixed

File: tests/CRUD/rooms.js (hunk 2 above)
Comment (BrayanRosas) [inline tests/CRUD/rooms.js]: It already was fixed.

File: tests/CRUD/rooms.js (hunk 3 above)
Comment (BrayanRosas) [inline tests/CRUD/rooms.js]: I change the variable's name of Json and Json2 to a name more descriptive.

File: tests/Smoke/rooms.js (hunk 6 above)
Comment (BrayanRosas) [inline tests/Smoke/rooms.js]: I remove all comment please check it.

**Pull request merged**